package.json

```diff
@@ -56,6 +56,15 @@
 		"onCommand:markdownlint.fixLine"
 	],
 	"contributes": {
+		"languages": [
```

Comment:
Please move this section below "jsonValidation" to (mostly) sort the non-"configuration" items. ("menus" is out of place, but that's my fault.)